src/Components/ArticleDetails.jsx

```diff
@@ -20,6 +21,7 @@ function ArticleDetails() {
       .then((response) => {
         setArticle(response.data.article);
         setVoteCount(response.data.article.votes);
+        setCommentCount(response.data.article.comment_count)
```

Comment:
rather than return the entire response object from your axios call, return what you need specifically i.e.
return response.data.article in you api.js function and then setArticle(article) etc. it is more readable like this